examples/messager/Cargo.toml

```diff
@@ -5,8 +5,14 @@ authors.workspace = true
 license.workspace = true
 edition.workspace = true
 
+[[bin]]
+name = "demo_messager"
```

Comment:
Maybe `messenger`?

https://english.stackexchange.com/questions/211365/why-is-messenger-the-term-instead-of-messager

Reply:
https://github.com/gear-tech/gear/issues/3621 will do this in another PR since this related to workspace changes